roles/contiv/templates/contiv.cfg.j2

```diff
@@ -0,0 +1,6 @@
+{
+  "K8S_API_SERVER": "https://{{ hostvars[groups['masters'][0]]['ansible_' + netmaster_interface].ipv4.address }}:{{ kube_master_api_port }}",
```

Comment:
openshift.master.cluster_hostname should point at a load balancer, whether that's an ELB, F5, VIP, or other load balancer in front of the API servers. Where as pointing at the ip address of one of the masters would never be load balanced.